itou/utils/validators.py

```diff
@@ -37,11 +37,12 @@ def validate_naf(naf):
 
 
 def validate_pole_emploi_id(pole_emploi_id):
-    is_valid = len(pole_emploi_id) == 8 and pole_emploi_id[:7].isdigit() and pole_emploi_id[7:].isalnum()
-    if not is_valid:
+    is_old_format = len(pole_emploi_id) == 8 and pole_emploi_id[:7].isdigit() and pole_emploi_id[7:].isalnum()
+    is_new_format = len(pole_emploi_id) == 11 and pole_emploi_id.isdigit()
+    if not (is_new_format or is_old_format):
         raise ValidationError(
-            "L'identifiant France Travail doit être composé de 8 caractères : "
-            "7 chiffres suivis d'une 1 lettre ou d'un chiffre."
+            "L’identifiant que vous avez renseigné n’est pas conforme "
+            f"car il comporte {len(pole_emploi_id)} caractères."
```

Comment:
Je trouve ça assez peu utile de juste indiquer la longueur plutôt que les formats attendus
```suggestion
            "L’identifiant France Travail doit être composé de 7 chiffres suivis d'une 1 lettre ou d'un chiffre, ou de 11 chiffres."
```
Ce qui devrais en plus permettre de ne pas avoir à faire `pole_emploi_id_error` et de pouvoir plus facilement réutiliser `validate_pole_emploi_id()`.

Reply:
Alors j'ai suivi ce qu'il y a écrit dans la carte.
Sachant que le formulaire indique déjà `L’identifiant France Travail doit être composé de 7 chiffres suivis d'une 1 lettre ou d'un chiffre, ou de 11 chiffres` en help text